lib/Sema/SemaBounds.cpp

```diff
@@ -909,6 +911,15 @@ namespace {
         }
       }
 
+      if (LHSType->isPointerType() &&
```

Comment:
I do not know why this code is there. I thought I deleted it. I checked my local repository and there is no this code. I will make sure that there is no this code in next PR.